infra/blueprint-test/pkg/krmt/krm.go

```diff
@@ -24,20 +24,21 @@ var CommonSetters = []string{"PROJECT_ID", "BILLING_ACCOUNT_ID", "ORG_ID"}
 
 // KRMBlueprintTest implements bpt.Blueprint and stores information associated with a KRM blueprint test.
 type KRMBlueprintTest struct {
-	name         string                   // descriptive name for the test
-	exampleDir   string                   // directory containing KRM blueprint example
-	buildDir     string                   // directory to hydrated blueprint configs pre apply
-	kpt          *kpt.CmdCfg              // kpt cmd config
-	timeout      string                   // timeout for KRM resource status
-	setters      map[string]string        // additional setters to populate
-	updatePkgs   bool                     // whether to update packages in exampleDir
-	updateCommit string                   // specific commit to update to
-	logger       *logger.Logger           // custom logger
-	t            testing.TB               // TestingT or TestingB
-	init         func(*assert.Assertions) // init function
-	apply        func(*assert.Assertions) // apply function
-	verify       func(*assert.Assertions) // verify function
-	teardown     func(*assert.Assertions) // teardown function
+	name         string                         // descriptive name for the test
+	exampleDir   string                         // directory containing KRM blueprint example
+	testConfig   *discovery.BlueprintTestConfig // additional blueprint test configs
```

Comment:
This should be embedded, not pointed to.

Reply:
done